news/2021-09-22-new-featured-artist-litmus.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: litmus\*"
```

Comment:
...ya i'm not sure whether you need this `\` escape character or not tbh since this is the title field (not the article's body)

someone might know better about this i suppose :(

Reply:
you don't need to escape this: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#titles